src/components/Menu.jsx

```diff
@@ -3,12 +3,6 @@ import { CSSTransition } from 'react-transition-group';
 import { stxtap } from '../store/utils';
 
 class Menu extends Component {
-    onOverlayClick = (e) => {
```

Comment:
@MahboobehMohammadi Why do you remove this part?

Reply:
@shayanbinary The dialog will be closed on `handleClickOutside` if user click outside of it and also it will be closed when another dialog opens, so this code realy never uses, and we certainly dont want the dialog to close when we click on it unless we tell it to. so this code can be safely  removed.

Reply:
@shayanbinary I just realized this had a use afterall. since $(document).click() not working correctly on iPhone, this event was using to close the open dialog when user click outside it, but since it was causing problems on android devices after i made the dialog to fill the screen, i just fixed that problem some other way. if you wanna know more check this:
https://stackoverflow.com/questions/3705937/document-click-not-working-correctly-on-iphone-jquery

Reply:
They still can!